flatland/contrib/training/flatland_pettingzoo_stable_baselines.py

```diff
@@ -1,127 +0,0 @@
-
-import numpy as np
-import os
-import PIL
-import shutil
-
-from stable_baselines3.ppo import MlpPolicy
-from stable_baselines3 import PPO
-
-import supersuit as ss
-
-from flatland.contrib.interface import flatland_env
-from flatland.contrib.utils import env_generators
-
-from flatland.envs.observations import TreeObsForRailEnv
-from flatland.envs.predictions import ShortestPathPredictorForRailEnv
-
-import fnmatch
-import wandb
-
-"""
-https://github.com/PettingZoo-Team/PettingZoo/blob/HEAD/tutorials/13_lines.py
-"""
-
-# Custom observation builder without predictor
-# observation_builder = GlobalObsForRailEnv()
-
-# Custom observation builder with predictor
-observation_builder = TreeObsForRailEnv(max_depth=2, predictor=ShortestPathPredictorForRailEnv(30))
-seed = 10
-np.random.seed(seed)
-wandb_log = False
-experiment_name = "flatland_pettingzoo"
-
-try:
-    if os.path.isdir(experiment_name):
-        shutil.rmtree(experiment_name)
-    os.mkdir(experiment_name)
-except OSError as e:
-    print("Error: %s - %s." % (e.filename, e.strerror))
-
-# rail_env = env_generators.sparse_env_small(seed, observation_builder)
-rail_env = env_generators.small_v0(seed, observation_builder)
-
-# __sphinx_doc_begin__
-
-env = flatland_env.parallel_env(environment=rail_env, use_renderer=False)
-# env = flatland_env.env(environment = rail_env, use_renderer = False)
-
-if wandb_log:
-    run = wandb.init(project="flatland2021", entity="nilabha2007", sync_tensorboard=True, 
-                     config={}, name=experiment_name, save_code=True)
```

Comment:
Illustration of logging/callbacks incl. `wandb` will be added again with #77 / #76